docs/guides/completions.md

```diff
@@ -41,34 +58,41 @@ An example Completion API response looks as follows:
     ```
 === "Response in python"
     ```python
-    >>> print( response.outputs[0].text )
-    ________. Hello, my name is _
+    >>> print(response.output.text)
+    ```
+    Example output:
+    ```python
+    _______ and I am a _______
     ```
 
 ## Token streaming
 
-The Completions API support token streaming to reduce _perceived_ latency for certain applications. When streaming,
-tokens will be sent as data-only [server-side events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#event_stream_format).
+The Completions API supports token streaming to reduce _perceived_ latency for certain 
+applications. When streaming, tokens will be sent as data-only 
+[server-side events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#event_stream_format).
 
 To enable token streaming, pass `stream=True` to either `Completion.create` or `Completion.acreate`.
 
 An example of token streaming using the synchronous Completions API looks as follows
 
 === "Token streaming with synchronous API in python"
 ```python
+import sys
+
 from llmengine import Completion
 
 stream = Completion.create(
-    model="llama-7b",
-    prompt="why is the sky blue?",
-    max_new_tokens=5,
+    model="falcon-7b-instruct",
+    prompt="Give me a 200 word summary on the current economic events in the US.",
+    max_new_tokens=1000,
     temperature=0.2,
-    stream=True,
+    stream=True
```

Comment:
nit: you can leave the comma if you want. I think it's idiomatic to leave the trailing commas in python

Reply:
Ah yes, copied this from the getting started page. Put it back